docs/sdks/stash/uid2-sdk-ref-mobile-ima.md

```diff
@@ -0,0 +1,113 @@
+# Developer Guide for UID2 IMA Plugin (iOS and Android)
+
+As of 2023-07-15
+
+## Introduction
+
+The UID2 IMA Plugin is pre-release software and should be used as such.
+
+The UID2 IMA Plugin enables apps that use Google's IMA SDK to include
+UID2 data in their ad requests. This is done automatically, with no
+direct coding needed from the app developer, by:
+
+-   The IMA SDK
+
+-   The UID2 SDK
+
+-   The UID2 IMA Plugin
+
+## iOS
+
```

Comment:
can remove line 21-27 as it's rhetorical

Reply:
Noted for IMA doc. Also -- GMA and IMA docs will both have separate versions for iOS and Android, and the heading structure will reflect that.